firmware/config.h

```diff
@@ -3,6 +3,7 @@
 
 enum {
 	CONFIG_KEY_RESOLUTION = 0,
+	CONFIG_KEY_RESOLUTION_SEC,
```

Comment:
RESOLUTION_SEC completes to RESOLUTION_SECONDS rather than RESOLUTION_SECONDARY...

Maybe CONFIG_KEY_RESOLUTION_2ND or change both to CONFIG_KEY_RES_PRIMARY and CONFIG_KEY_RES_SECONDARY?